contributors/devel/testing.md

```diff
@@ -215,7 +215,7 @@ script to run a specific integration test case:
 
 ```sh
 # Run integration test TestPodUpdateActiveDeadlineSeconds with the verbose flag set.
-make test-integration WHAT=./test/integration/pods KUBE_GOFLAGS="-v" KUBE_TEST_ARGS="-run ^TestPodUpdateActiveDeadlineSeconds$"
+make test-integration WHAT=pods KUBE_GOFLAGS="-v" KUBE_TEST_ARGS="-run ^TestPodUpdateActiveDeadlineSeconds$"
```

Comment:
The issue got fixed part of https://github.com/kubernetes/kubernetes/commit/98f5c362ff11ade66788f47630b9c9b4abacd704, hence closing this PR.